stockApp/stockApp/Presentation/View/AssetTableViewCell.swift

```diff
@@ -0,0 +1,184 @@
+    static let backViewTop: Int = 10
+    static let backViewLeading: Int = 0
+    static let backViewTrailing: Int = 0
+    static let backViewBottom: Int = -10
+    static let barcodeTop: Int = 20
+    static let barcodeLeading: Int = 10
+    static let barcodeTrailing: Int = 0
+    static let assetNameTop: Int = 7
+    static let assetNameLeading: Int = 10
+    static let assetNameTrailing: Int = 0
+    static let globerMailTop: Int = 7
+    static let globerMailLeading: Int = 10
+    static let globerMailTrailing: Int = 0
+    static let siteTop: Int = 7
+    static let siteLeading: Int = 10
+    static let siteTrailing: Int = 0
+    static let imageWidthMultiplier: Int = 0
+    static let imageWidthConstant: Int = 20
+    static let imageTop: Int = 70
+    static let imageTrailing: Int = -10
+    static let imageBottom: Int = -70
+}
+
+// MARK: - Class
+
+class AssetTableViewCell: UITableViewCell {
+
+    // MARK: - Properties
+
+    let backView: UIView = {
+        let view = UIView()
+        view.backgroundColor = UIColor.black.withAlphaComponent(CGFloat(Constants.backViewAlpha))
+        return view
+    }()
+
+    let barcodeUILabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont(name: Strings.heeboFont, size: CGFloat(Constants.fontSize))
+        label.font = label.font.withSize(CGFloat(Constants.fontSizeLabel))
+        label.lineBreakMode = .byWordWrapping
+        label.numberOfLines = Constants.numberOfLines
+        label.textColor = Color.porcelainWhite
+        label.textAlignment = .left
+        return label
+    }()
+
+    let assetNameUILabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont(name: Strings.heeboFont, size: CGFloat(Constants.fontSize))
+        label.font = label.font.withSize(CGFloat(Constants.fontSizeLabel))
+        label.lineBreakMode = .byWordWrapping
+        label.numberOfLines = Constants.numberOfLines
+        label.textColor = Color.porcelainWhite
+        label.textAlignment = .left
+        return label
+    }()
+
+    let globerMailUILabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont(name: Strings.heeboFont, size: CGFloat(Constants.fontSize))
+        label.font = label.font.withSize(CGFloat(Constants.fontSizeLabel))
+        label.lineBreakMode = .byWordWrapping
+        label.numberOfLines = Constants.numberOfLines
+        label.textColor = Color.porcelainWhite
+        label.textAlignment = .left
+        return label
+    }()
+
+    let siteUILabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont(name: Strings.heeboFont, size: CGFloat(Constants.fontSize))
+        label.font = label.font.withSize(CGFloat(Constants.fontSizeLabel))
+        label.lineBreakMode = .byWordWrapping
+        label.numberOfLines = Constants.numberOfLines
+        label.textColor = Color.porcelainWhite
+        label.textAlignment = .left
+        return label
+    }()
+
+    let imageUIImageView: UIImageView = {
```

Comment:
```suggestion
    let arrowImageView: UIImageView = {
```
give context to the name!

Reply:
done